src/cordova/android/OutlineAndroidLib/outline/src/main/java/org/outline/vpn/VpnTunnelService.java

```diff
@@ -30,627 +30,634 @@
 import android.net.VpnService;
```

Comment:
Could you please give an overview of what's changing in this file?

Also, please restore the 2-space indentation, so we can see the changes and for consistency.

Reply:
Sorry, my bad.

Here is what I've changed,
```
private void tearDownActiveTunnel() {
    stopVpnTunnel();
    stopForeground();
    // add this line, line 361
    broadcastVpnConnectivityChange(TunnelStatus.DISCONNECTED);
    tunnelConfig = null;
    stopNetworkConnectivityMonitor();
    tunnelStore.setTunnelStatus(TunnelStatus.DISCONNECTED);
  }
```